_dev/src/components/commons/banner-ads.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <b-card
+    class="border-0"
+    :class="{ 'shadow-sm mb-3' : size !== 'xs' }"
+  >
+    <button
+      v-if="isBanner"
+      type="button"
+      class="close"
+      aria-label="Close"
+      @click="closeBanner"
+    >
+      <span aria-hidden="true">&times;</span>
+    </button>
+    <div class="d-flex align-items-center">
+      <img
+        :src="require(`@/assets/images/${pngBanner}`)"
+        alt="banner-price"
+        :height="size === 'xs' ? 90 : 140"
+        :width="size === 'xs' ? 90 : 140"
+        class="mr-4"
+      >
+      <span>
+        <h3 class="font-weight-600">
+          {{ $t("banner.titleAdsbanner") }}
+        </h3>
+
+        <p>
+          {{ $t("banner.textAdsBanner", [$t("banner.priceAdsBanner")]) }}
+        </p>
+        <p class="ps_gs-fz-10">
+          {{ $t("banner.legendConfigured") }}
+        </p>
+      </span>
+
+      <img
```

Comment:
Should probably a displayed using `background-image` to not bloat the markup.  
What do you think ?

Reply:
Instead of creating a new scss file just for this background, i stay with this <img> but remove it on mobile